app/src/main/java/com/babylon/wallet/android/presentation/settings/linkedconnectors/LinkedConnectorsScreen.kt

```diff
@@ -228,48 +268,104 @@ private fun ActiveLinkedConnectorsListContent(
 
 @Composable
 private fun ActiveLinkedConnectorContent(
-    activeLinkedConnector: LinkedConnectorsUiState.ConnectorUiItem,
+    activeLinkedConnector: ConnectorUiItem,
     modifier: Modifier = Modifier,
-    onDeleteConnectorClick: (PublicKeyHash) -> Unit,
+    onRenameConnectorClick: (connectorUiItem: ConnectorUiItem) -> Unit,
+    onDeleteConnectorClick: (id: PublicKeyHash) -> Unit,
 ) {
-    Column(modifier = modifier) {
+    Column(modifier = modifier.background(color = RadixTheme.colors.white)) {
         Row(
             Modifier
                 .fillMaxWidth()
                 .padding(horizontal = RadixTheme.dimensions.paddingDefault),
-            horizontalArrangement = Arrangement.spacedBy(RadixTheme.dimensions.paddingSmall),
             verticalAlignment = Alignment.CenterVertically
         ) {
             Text(
                 modifier = Modifier.weight(1f),
                 text = activeLinkedConnector.name,
                 style = RadixTheme.typography.body2Regular,
-                color = RadixTheme.colors.gray2
+                color = RadixTheme.colors.gray2,
+                maxLines = 1,
+                overflow = TextOverflow.Ellipsis
             )
+            IconButton(
```

Comment:
The spacing between the 2 icon buttons is larger than what's in Zeplin. I think this is because of `Modifier.minimumInteractiveComponentSize` from inside `IconButton` implementation. We can easily override the size to match Zeplin.

Reply:
yup but there is a reason that the `minimumInteractiveComponentSize` exists, and that's because of accessibility. 
Wouldn't be bad idea to break this min size?